packages/alice_objectbox/pubspec.yaml

```diff
@@ -11,13 +11,14 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  alice: ^1.0.0-dev.6
+  alice: ^1.0.0-dev.8
   json_annotation: ^4.9.0
   meta: ^1.9.1
   objectbox: ^4.0.1
   objectbox_flutter_libs: ^4.0.1
   path: ^1.9.0
   path_provider: ^2.1.3
+  flat_buffers: ^23.5.26
```

Comment:
Not sure why this is need. This is a default transitive dependency and should probably not be directly depended on.

Reply:
Ah I see, the generated file references it directly
```
Package validation found the following error:
* line 11, column 1 of lib/objectbox.g.dart: This package does not have flat_buffers in the `dependencies` section of `pubspec.yaml`.
     ╷
  11 │ import 'package:flat_buffers/flat_buffers.dart' as fb;
     │ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
     ╵
```

Reply:
Exactly :)